tests/forms_tests/widget_tests/test_selectdatewidget.py

```diff
@@ -347,9 +347,6 @@ def test_selectdate_required_placeholder(self):
                     'days_options': '\n'.join(
                         '<option value="%s">%s</option>' % (i, i) for i in range(1, 32)
                     ),
-                    'm_placeholder': 'placeholder="Month"' if required else '',
-                    'd_placeholder': 'placeholder="Day"' if required else '',
-                    'y_placeholder': 'placeholder="Year"' if required else '',
                     'empty': '' if required else '<option selected value="">---</option>',
```

Comment:
This line looks backwards... (If I add a `required` to the select, the HTML validator will trigger on the lack of the empty value.)